### PR TITLE
Add Tyk OSS Path to Custom Plugin Get Started Tutorial

### PR DESCRIPTION
**Pull request by sedkis**
1. Updated the instructions as per the improvements from 2 weeks ago
2. Added Tyk OSS path for the tutorial.

These changes to be released with the Tyk OSS path added in the [get started repo](https://github.com/TykTechnologies/custom-go-plugin/pull/11)

review page https://deploy-preview-2338--tyk-docs.netlify.app/docs/nightly/plugins/get-started-selfmanaged/get-started/

### CODE REVIEW

File: tyk-docs/content/plugins/get-started-selfmanaged/get-started.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 date: 2017-03-24T15:45:13Z
 title: Get Started with Custom Plugins
+tags: ["custom", "plugin", "plugins", "go", "goplugins"]
```
Comment (letzya) [inline tyk-docs/content/plugins/get-started-selfmanaged/get-started.md]:
```suggestion
tags: ["custom", "plugin", "plugins", "go", "goplugins", "go plugin", "tyk go plugin", "golang plugin"]
```

File: tyk-docs/content/plugins/get-started-selfmanaged/run.md [hunk 2]
```diff
@@ -7,68 +7,63 @@ menu:
...
 ---
 
-We have a getting started repo to help you set up your development environment.
-
-### 1.  Clone the getting started repo
-
-Please clone the [getting started repo][0].
-
-
+We'll be using Tyk's getting started repo to set up your development environment.
```
Comment (letzya) [inline tyk-docs/content/plugins/get-started-selfmanaged/run.md]:
```suggestion
We'll be using Tyk's getting started repo, https://github.com/TykTechnologies/custom-go-plugin, to get you up and running with a fully working Tyk custom go plugin.
```

File: tyk-docs/content/plugins/get-started-selfmanaged/test.md [hunk 3]
```diff
@@ -43,89 +47,86 @@ Yields the response:
...
   "args": {},
   "headers": {
     "Accept": "*/*",
-    "Host": "localhost:8000",
+    "Accept-Encoding": "gzip",
+    "Foo": "Bar",
+    "Host": "httpbin.org",
     "User-Agent": "curl/7.79.1",
-    "Foo": "Bar"
+    "X-Amzn-Trace-Id": "Root=1-63f792ce-3d57c23571c67fc46b70539c"
```
Comment (letzya) [inline tyk-docs/content/plugins/get-started-selfmanaged/test.md]:
```suggestion
```

File: tyk-docs/content/plugins/get-started-selfmanaged/test.md [hunk 4]
```diff
@@ -43,89 +47,86 @@ Yields the response:
...
   },
-  "origin": "172.17.0.1",
-  "url": "http://localhost:8000/get"
+  "origin": "172.28.0.1, 99.242.70.243",
+  "url": "http://httpbin.org/get"
 }
 ```
 
-Awesome!  Our Go plugin was executed, as evident in the echo response from our default API above which contains the newly added Header.
+We've sent an API request to the Gateway.   We can see that the sample custom plugin has injected an HTTP Header with a value of "Foo:Bar".  This header was echo'd back to us in the Response Body by the mock Httpbin server.
```
Comment (letzya) [inline tyk-docs/content/plugins/get-started-selfmanaged/test.md]:
```suggestion
We've sent an API request to the Gateway. We can see that the sample custom plugin has injected an HTTP Header with a value of "Foo:Bar". This header was echoed back to us in the Response Body by the mock Httpbin server.
```

File: tyk-docs/content/plugins/get-started-selfmanaged/run.md [hunk 5]
```diff
@@ -7,68 +7,63 @@ menu:
...
 
+This will take a few minutes to run as it compiles the plugin for the first time and downloads all the necessary Docker images.
+
 {{< tab_end >}}
 {{< tab_start "Open Source" >}}
 
-run the `make` command:
+### 1.  Clone the getting started repo
+
+Please clone the [getting started repo][0].
```
Comment (letzya) [inline tyk-docs/content/plugins/get-started-selfmanaged/run.md]:
```suggestion
Please clone the getting started repo, [custom-go-plugin](https://github.com/TykTechnologies/custom-go-plugin).
```

Comment (letzya) [reply tyk-docs/content/plugins/get-started-selfmanaged/run.md]: please delete the last line  no.76 .

**Comment by dle519** [comment]
Following the instructions was very quick and very simple and easy to do!

However, docker changed the way the names of the containers are generated causing the "docker logs custom-go-plugin_tyk-pump_1" command to not work but it can be simply fixed if you replace the container name with its container ID!

File: tyk-docs/content/plugins/supported-languages/golang.md [hunk 6]
```diff
@@ -11,7 +11,12 @@ aliases:
   - /plugins/supported-languages/golang/
 ---
 
-This guide will help you to understand the idea behind Golang plugins in less than 5-10 minutes.
+
+
+This is an advanced guide to help you understand more in-depth cncepts behind Golang plugins.
```
Comment (dle519) [inline tyk-docs/content/plugins/supported-languages/golang.md:16]: spelling mistake

**Review by dle519 — APPROVED** [review]
looks amazing to me

**Pull request merged**